recipes/vsclust/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "VSClust" %}
+{% set version = "0.85" %}
+
+package:
+  name: vsclust
+  version: {{ version }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+source:
+  url: https://bitbucket.org/veitveit/vsclust/get/release-0.85.tar.gz
+  sha256: 481b419f3bb0d72c8469bfb04acfff227faf11c73ec66f83668edfc95ad94268
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+  host:
+    - r-base 3.6.3
+    - r-matrixstats >=0.56
+    - bioconductor-mfuzz >=2.46.0
+    - bioconductor-qvalue >=2.16.0
+    - bioconductor-limma
+  run:
+    - r-base
+    - r-matrixstats >=0.56.0
+    - bioconductor-mfuzz >=2.46.0
+    - bioconductor-qvalue >=2.16.0
+    - bioconductor-limma
+    - bioconductor-clusterprofiler 
+    - bioconductor-RDAVIDWebService
+    - r-readxl >=1.3.1
+    - r-shiny >=1.4.0
+    - r-yaml >=2.2.0
+    - r-shinyjs >=1.1
+    - r-shinythemes >=1.1.2
+    - r-data.table >=1.12.8
+    
+test:
+  commands:
+     - cd $PREFIX/bin; runVSClust.R vsclust.yml; cd -
```

Comment:
You should not need to change directories into the installation directory.

Reply:
Thanks, I did not know that. This should be solved now.